README.md

```diff
@@ -51,7 +51,37 @@ duti -s org.dandavison.OpenInEditor file-line-column
 ```
 
 ### Linux
-TODO
+
+On a system that complies to the [XDG shared MIME-info DB specification](https://specifications.freedesktop.org/shared-mime-info-spec/shared-mime-info-spec-latest.html#idm140625828587776), there are two simple steps to follow. This should apply to the majority of current GNU/Linux installations - if you're unsure check if `type -P xdg-mime` returns a file path.
+
+First, you need to create a `.desktop` file which references `open-in-editor`, and then set that as default for either the `file-line-column://` or the `file://` URL scheme, depending on your choice. For example execute the following commands in a shell :
+
+```sh
+$ cat >~/.local/share/applications/augmented-open.desktop <<OVER
+[Desktop Entry]
+Type=Application
+Name=AugmentedOpen
+GenericName=Open a file at a certain position
+Comment=Opens URLs of the type file-line-column://<path>[:<line>[:<column>]] in the configured editor and positions the cursor
+Icon=text-editor
+Exec=open-in-editor %U
+Categories=Utility;Core;
+StartupNotify=false
+MimeType=x-scheme-handler/file-line-column
+OVER
+
+$ xdg-mime default augmented-open.desktop x-scheme-handler/file-line-column
```

Comment:
Does the second argument need to resolve to the file on disk? I.e. would it be clearer to write this instruction like this?

```
xdg-mime default ~/.local/share/applications/augmented-open.desktop x-scheme-handler/file-line-column
```

Reply:
No, XDG defines a search path for this. ~/.local/share/applications/ is part of it and overrides files of the same name in directories which are earlier in that search path. E.g. ~/.local/share/applications/appX.desktop has priority over /usr/share/applications/appX.desktop if both exist.

Reply:
**PS** This argument repesents the "[desktop file ID](https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html)" (i.e. not a file path).